runbotics/runbotics-scheduler/src/websocket/process-launch/bot-process-event.service.ts

```diff
@@ -72,16 +69,17 @@ export class BotProcessEventService {
                 .execute();
             await this.updateProcessInstance(queryRunner, processInstanceEvent);
             await queryRunner.commitTransaction();
-
             const updatedProcessInstanceEvent =
                 await this.processInstanceEventService.findByExecutionId(
                     queryRunner,
                     processInstanceEvent.executionId
                 );
-
             if (
                 updatedProcessInstanceEvent.processInstance
-                    .rootProcessInstanceId === null
+                    .rootProcessInstanceId === null && 
+                    !(processInstanceEvent.status === ProcessInstanceEventStatus.IN_PROGRESS &&
+                        updatedProcessInstanceEvent.status === ProcessInstanceEventStatus.COMPLETED
+                    ) 
```

Comment:
here should be error status included imo

Reply:
trueeee